CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
+ * Fix test with markdown on Windows OS. See [lsp4xm#487](https://github.com/angelozerr/lsp4xml/issues/487)
+ * Fixed case sensitivity problems for element and attribute names. See [lsp4xm#433](https://github.com/angelozerr/lsp4xml/issues/433)
+ * Selection formatting ignores attribute indentation preference. See [lsp4xm#429](https://github.com/angelozerr/lsp4xml/issues/429) 
+ * Fixed error range for `src-import.1.2`. See [lsp4xm#499](https://github.com/angelozerr/lsp4xml/issues/499)
+ * Fixed error range for `s4s-elt-invalid-content.3`. See [lsp4xm#496](https://github.com/angelozerr/lsp4xml/issues/496)
+ * Fixed error range for `EntityNotDeclared`. See [lsp4xm#518](https://github.com/angelozerr/lsp4xml/issues/518)
+ * Fixed error range for `cvc-pattern-valid`. See [lsp4xm#477](https://github.com/angelozerr/lsp4xml/issues/477)
+ * Fixed error range for `AttributePrefixUnbound`. See [lsp4xm#476](https://github.com/angelozerr/lsp4xml/issues/476)
+ * Fixed error range for `EmptyTargetNamespace`. See [lsp4xm#472](https://github.com/angelozerr/lsp4xml/issues/472)
+ * Fixed error range for `ct-props-correct.3`. See [lsp4xm#467](https://github.com/angelozerr/lsp4xml/issues/467)
+ * Fixed error range for `sch-props-correct.2`. See [lsp4xm#462](https://github.com/angelozerr/lsp4xml/issues/462)
+ * Fixed error range for `s4s-elt-must-match.2`. See [lsp4xm#458](https://github.com/angelozerr/lsp4xml/issues/458)
+ * Fixed error range for `ct-props-correct.3`. See [lsp4xm#455](https://github.com/angelozerr/lsp4xml/issues/455)
+ * Fixed error range for `src-ct.1`. See [lsp4xm#453](https://github.com/angelozerr/lsp4xml/issues/453)
+ * Fixed error range for `duplicate attribute`. See [lsp4xm#452](https://github.com/angelozerr/lsp4xml/issues/452)
+ * Fixed error range for `p-props-correct.2.1`. See [lsp4xm#436](https://github.com/angelozerr/lsp4xml/issues/436)
+ * Fixed error range for `cos-all-limited.2`. See [lsp4xm#428](https://github.com/angelozerr/lsp4xml/issues/428)
+ * Fixed error range for `src-element.3`. See [lsp4xm#420](https://github.com/angelozerr/lsp4xml/issues/420)
+ * Documents with an Internal Subset DOCTYPE had stopped trying to bind. See [lsp4xm#379](https://github.com/angelozerr/lsp4xml/issues/379)
+ * XML did not validate when bounded DTD file was not found. See [#167](https://github.com/redhat-developer/vscode-xml/issues/167)
+ * Unexpected indentation after empty element and attribute. See [#159](https://github.com/redhat-developer/vscode-xml/issues/159)
+ * Formatter inserts spaces in empty lines. See [lsp4xm#157](https://github.com/redhat-developer/vscode-xml/issues/157)
+ * Plugin is not recognized as a XML formatter. See [#154](https://github.com/redhat-developer/vscode-xml/issues/154)
+ * VSCode was not revalidating XML files when relevant XSD files were modified outside VSCode. See [#131](https://github.com/redhat-developer/vscode-xml/issues/131)
```

Comment:
either this one or enhancement 132 should be removed, as one is a duplicate of the other

Reply:
@fbricon updated